portal-impl/src/com/liferay/portlet/messageboards/service/impl/MBMessageLocalServiceImpl.java

```diff
@@ -2602,9 +2602,9 @@ private String _getLocalizedCategoryName(
 				group.getDescriptiveName(locale);
 		}
 		catch (PortalException pe) {
-			_log.error(
-				"Could not retrieve group name for {groupId=" +
-					category.getGroupId() + "}");
+			if (_log.isWarnEnabled()) {
```

Comment:
Hey @robertoDiaz can we make this logging same level as the others (error) ? I think that they should all be consistent in the level log.